docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx

```diff
@@ -62,3 +62,28 @@ If you have a server with no firewall, there is nothing to be done, otherwise ma
 
 On the desktop side if you have a router in front of your computer, you'll need to forward TCP and UDP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](../Additional-Guides/port-forwarding) for a more in-depth tutorial).
 If you're connected directly without any router, then again nothing needs to be done in such case.
+
+### Storage
+
+**HDDs are not supported and will never be**. We don't recommended trying to use smart caching, tiered storage or other ways to accelerate it, you'll be 100% disappointed and just waste your time.
+
+**Node will require 100 GiB of good quality SSD**. Doesn't have to be anything amazing, but something mid-range from a
+reputable manufacturer with decent endurance is recommended.
```

Comment:
Why does this line have a break in it? This will make it render oddly at different resolutions?

Reply:
I'm not sure tbh, the line was there to make it easier to look at the text via VSCode. Removed it to be certain that it renders fine.

Reply:
Word wrap in VSCode rather than setting line lengths is the way :)